docs/conceptual/cmake-packages.rst

```diff
@@ -98,6 +98,10 @@ there's a catch-all, last resort variable consulted locating this file,
 ``-D CMAKE_HIP_COMPILER_ROCM_ROOT:PATH=`` which should be set the root of the
 ROCm installation.
 
+.. note::
+    Imported targets defined by `hip-lang-config.cmake` are for internal use
```

Comment:
@MathiasMagnus what is the objective of this note? How would it help external users to know that this is only for internal consumption? How will our users benefit?